projects/plutus/inc/ltc2484.h

```diff
@@ -44,6 +44,11 @@ typedef union {
   int32_t i32data;
 } Ltc2484Response;
 
-// Parse the raw 24-bit ADC reading to a voltage measurement in uV. This is
-// called by ltc_adc_read, and is exposed for testing
+
+// The following definitions are exposed for testing
+
+// An arbitray voltage for testing
+#define TEST_INPUT_VOLTAGE 1000
```

Comment:
Don't define test things in the header. Why are you hardcoding this value?

If you want to use an x86 stub, just build it to support setting a value.

Reply:
Fixed